frontend/src/app/home/home.component.ts

```diff
@@ -46,7 +46,7 @@ export class HomeComponent implements OnInit {
   private _studiesToNewsItems(studies: Study[]): NewsItem[] {
     if (this.currentStudies && this.currentStudies.length > 0) {
       return studies
-        .sort((a, b) => (a.id > b.id) ? 1 : -1)
+        .sort((a, b) => (a.id < b.id) ? 1 : -1)
```

Comment:
I opened issue #436 because I'd love to get rid of this sorting/filtering on the front end. At the moment I don't thin that the performance is taking too much of a hit here, but we can switch endpoints and follow the lead of the studies page to clean this up a bit more. For now this fixes the issue and works great.